CHANGES.txt

```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Made `sys.nodes` queries resilient. In case if one of the nodes is not
```

Comment:
Think this should be rephrased somehow. E.g. something like

> Queries on the `sys.nodes` table will now short circuit in case some nodes in the cluster are unresponsive.
   All columns of the unresponsive nodes will  contain null values except `id` and `name`.